packages/integrations/alpinejs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@astrojs/alpinejs",
-  "description": "The official Alpine.js integration for Astro.",
+  "description": "Use Alpine within Astro",
```

Comment:
"Within" sounds a little formal here. Is there another word that's accurate enough and doesn't sound weird? "with" or "in"... :laughing:

Reply:
I think "with" is fine, I used "within" just to be consistent with the other UI framework integration packages, they all follow "Use (React) components within Astro". The "components" part doesn't apply to Alpine because it's more of additional methods and properties to use in HTML.

Reply:
@Yan-Thomas I am generally happier using "with" unless you really mean to emphasize *inside*. The place I mostly use "within" in docs is to say that something has to be located "within" `src/pages/` (meaning, anywhere inside, including a subdirectory.)  I could also see using it "within a component..." or "within your template..." again, meaning inside.

So if you want to update others to say "with" I think that's great! I also think if people are searching, they're more likely to type "How do I use Alpine with Astro" (or "and Astro") not "within."